<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventControlWidget</class>
 <widget class="QWidget" name="EventControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <widget class="QLineEdit" name="descriptionLineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="LineEditWithUnicodeDialog" name="nameLineEdit" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="timeLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Label deviation</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="labelDeviationLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Pen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="AnyComboBox" name="penStyleComboBox" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="penWidthSpinBox">
         <property name="singleStep">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ColorChoiseWidget" name="penColorChoiceWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Brush</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="ColorChoiseWidget" name="brushColorWidget" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Brush style</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Brush basis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="AnyComboBox" name="brushStyleComboBox" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="AnyComboBox" name="brushBasisComboBox" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Brush duration</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="brushDurationLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addArrowedTextPushButton">
     <property name="text">
      <string>Add arrowed text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/anycombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditWithUnicodeDialog</class>
   <extends>QWidget</extends>
   <header>commonWidgets/lineeditwithunicodedialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorChoiseWidget</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/colorchoisewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
